src/main/kotlin/org/rust/cargo/runconfig/command/CargoCommandConfiguration.kt

```diff
@@ -43,6 +43,7 @@ class CargoCommandConfiguration(
     var channel: RustChannel = RustChannel.DEFAULT
     var command: String = "run"
     var nocapture: Boolean = true
+    var exact: Boolean = false
```

Comment:
I think it wold be better not to surface this flag up to `CargoCommandConfiguration`, and instead just add it as an `additionalArgument`: we try to keep the number for various checkboxes for Cargo run configuration as small as possible :)